_posts/2022-09-28-ltr-with-opensearch-and-metarank.markdown

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title:  "Learn-to-Rank with OpenSearch and Metarank"
+authors:
+  - shuttie
+date:   2022-09-28 15:00:00 +0100
+categories:
+  - community
+---
+[Metarank](https://metarank.ai) is an open-source secondary ranker that can perform personalized reordering of search results with a [LambdaMART](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/MSR-TR-2010-82.pdf) Learn-to-Rank model. In this article, we'll discuss why & when LTR approach to ranking may be helpful and how Metarank implements it on top of OpenSearch. 
+
+## Ranking in Lucene and OpenSearch
```

Comment:
I think it may be a bit much to ask Roman to add about an OpenSearch specific feature. I am going to see if I can get a blurb together that does this.

Reply:
I made an update to the text, so it's now mentioning a per-shard re-scoring.